ChangeLogger.Action.UnitTests/Shared/GitHubActionOutputWriterTests.cs

```diff
@@ -0,0 +1,88 @@
+using ChangeLogger.Action.Shared;
+
+namespace ChangeLogger.Action.UnitTests.Shared;
+
+public class GitHubActionOutputWriterTests
+{
+    [Fact]
+    public void WriteReleaseNotes_WhenGitHubOutputNotSet_DoesNothing()
+    {
+        // arrange
+        Environment.SetEnvironmentVariable("GITHUB_OUTPUT", null);
+        var sut = new GitHubActionOutputWriter();
```

Comment:
Change `sut` to a class-level readonly variable, and initialise it in the constructor. Then update all the tests to use that.